pkg/virtctl/templates/templates.go

```diff
@@ -39,3 +46,15 @@ func OptionsUsageTemplate() string {
 {{.InheritedFlags.FlagUsages | trimTrailingWhitespaces}}{{end}}
 `
 }
+
+// ExactArgs validate the number of input parameters
+func ExactArgs(nameOfCommand string, n int) cobra.PositionalArgs {
+	return func(cmd *cobra.Command, args []string) error {
+		if len(args) != n {
+			fmt.Printf("fatal: Number of input parameters is incorrect, %s accepts %d arg(s), received %d\n\n\n\n", nameOfCommand, n, len(args))
```

Comment:
Why the 4 `\n`?

Reply:
It's because of the message format printed in the screen when appending the help directly after.
we need to distinguish between the error message and the help menu and when dropping the line 3 times and then printing the help menu it looks much better

Reply:
@jean-edouard  fixed